README.md

```diff
@@ -39,13 +36,14 @@ Official Helm charts to deploy Sombra and related services into a Kubernetes clu
         alb.ingress.kubernetes.io/subnets: <VPC_PUBLIC_SUBNET>
         alb.ingress.kubernetes.io/tags: env=dev
         alb.ingress.kubernetes.io/target-type: ip
-      host: sombra-transcend.my-domain.com
+      hosts:
+        - host: sombra-transcend.my-domain.com
```

Comment:
Can `my-domain` be a reference to a global variable? So that the customer only needs to change it at one spot